sentry_sdk/api.py

```diff
@@ -130,7 +130,7 @@ def configure_scope(  # noqa: F811
 
 
 @overload
-def push_scope():  # noqa: F811
```

Comment:
I fixed pyflakes a while back to understand `@overload`

Reply:
👍